src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -11414,9 +11415,10 @@ class Test
     }
 }
 ";
-            await VerifyItemIsAbsentAsync(source, "M0");
+            await VerifyItemExistsAsync(source, "M0");
```

Comment:
> await VerifyItemExistsAsync(source, "M0"); [](http://example.com/codeflow?start=11&length=43)

@Youssef1313, @CyrusNajmabadi This change is not expected. Only abstract  or virtual static members can be accessed on a type parameter.

Also, it would be good to include some (virtual/abstract/sealed) instance members to the interfaces and observe that they are not offered.

Reply:
→ 64733